launch_ros/launch_ros/actions/node.py

```diff
@@ -187,10 +182,7 @@ def node_name(self):
     def _create_params_file_from_dict(self, params):
         with NamedTemporaryFile(mode='w', prefix='launch_params_', delete=False) as h:
             param_file_path = h.name
-            # TODO(dhood): clean up generated parameter files.
```

Comment:
Is this TODO still relevant? I.e. Does the generated parameter file get cleaned up?

Reply:
You're absolutely right. Re-added.